api/cpp/esp-idf/slint/CMakeLists.txt

```diff
@@ -26,19 +28,27 @@ set(SLINT_LIBRARY_CARGO_FLAGS "-Zbuild-std=core,alloc")
 set(DEFAULT_SLINT_EMBED_RESOURCES "embed-for-software-renderer" CACHE STRING "")
 set(CMAKE_BUILD_TYPE Release)
 set(BUILD_SHARED_LIBS OFF)
-
+set(Rust_CARGO_TARGET ${rust_target})
 
 if (SLINT_ESP_LOCAL_EXAMPLE)
     add_subdirectory(../.. slint_build)
-else(SLINT_ESP_LOCAL_EXAMPLE)
-    include(FetchContent)
-    FetchContent_Declare(
-        Slint
-        GIT_REPOSITORY https://github.com/slint-ui/slint
-        GIT_TAG v1.8.0
-        SOURCE_SUBDIR api/cpp
-    )
-    FetchContent_MakeAvailable(Slint)
-endif(SLINT_ESP_LOCAL_EXAMPLE)
+else()
+    list(PREPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}/cmake")
+    set(SLINT_USE_NIGHTLY_VERSION TRUE) # REMOVED BY PACKAGING SCRIPT
+    find_package(Slint 1.8.0)
+
+    if (TARGET Slint::Slint)
+        set_target_properties(Slint::Slint PROPERTIES IMPORTED_GLOBAL TRUE)
+    else()
+        include(FetchContent)
+        FetchContent_Declare(
+            Slint
+            GIT_REPOSITORY https://github.com/slint-ui/slint
+            GIT_TAG master
```

Comment:
Should we not keep the expected version tag here?
(Otherwise we can also remove the line from upgrade_version.yaml)

But actually, why not put that in FindSlint.cmake?

Reply:
Good point about the version. Somehow I missed that. I'm torn about putting it into FindSlint.cmake - would prefer to leave it out for now.